scripts/openroad/read_write.tcl

```diff
@@ -0,0 +1,16 @@
+# Copyright 2020-2022 Efabless Corporation
```

Comment:
This is an exact duplicate of scripts/openroad/write_views.tcl

Reply:
removed